Day-II-Calculator/calculator/src/App.js

```diff
@@ -1,15 +1,78 @@
 import React from 'react';
 import './App.css';
-import ButtonContainer from './components/ButtonComponents/ButtonContainer.js';
+import CalcButton from './components/ButtonComponents/CalcButton.js';
 import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay.js';
 
-const App = () => {
-  return (
-    <div className="calc-container">
-      <CalculatorDisplay />
-      <ButtonContainer />
-    </div>
-  );
-};
+
+// const App = () => {
+//   return (
+//     <div className="calc-container">
+//       <CalculatorDisplay content='0' />
+//       <div className="button-container">
+//             <CalcButton buttonStyle='long-btn' text='clear' />
+//             <CalcButton buttonStyle='operator-btn' text='÷' />
+//             <CalcButton buttonStyle='number-btn' text='7' />
+//             <CalcButton buttonStyle='number-btn' text='8' />
+//             <CalcButton buttonStyle='number-btn' text='9' />
+//             <CalcButton buttonStyle='operator-btn' text='×' />
+//             <CalcButton buttonStyle='number-btn' text='4' />
+//             <CalcButton buttonStyle='number-btn' text='5' />
+//             <CalcButton buttonStyle='number-btn' text='6' />
+//             <CalcButton buttonStyle='operator-btn' text='-' />
+//             <CalcButton buttonStyle='number-btn' text='1' />
+//             <CalcButton buttonStyle='number-btn' text='2' />
+//             <CalcButton buttonStyle='number-btn' text='3' />
+//             <CalcButton buttonStyle='operator-btn' text='+' />
+//             <CalcButton buttonStyle='long-btn' text='0' />
+//             <CalcButton buttonStyle='operator-btn' text='=' />
+//         </div>
+//     </div>
+//   );
+// };
+
+class App extends React.Component {
+  constructor(props){
+    super(props);
+    this.state = {total: 0, operation: ''}
+  }
+
+  changeContent = (number) => {
+    this.setState({total: number})
```

Comment:
Whenever you are passing data it is good to check if what you are receiving is what you are intended.